java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -199,6 +199,9 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
                 .pfmerge(hllKey3, new String[] {hllKey1, hllKey2})
                 .pfcount(new String[] {hllKey3});
 
+        baseTransaction.append(key1, value1);
```

Comment:
can you move this test up to just after the strlen command (line 58)?  That way its the same test in PYthon and Java.